code/modules/admin/verbs/secrets.dm

```diff
@@ -203,17 +203,24 @@ GLOBAL_DATUM(everyone_a_traitor, /datum/everyone_is_a_traitor_controller)
 			if(!is_funmin)
 				return
 			SSblackbox.record_feedback("nested tally", "admin_secrets_fun_used", 1, list("Virus Outbreak"))
-			switch(tgui_alert(usr,"Do you want this to be a random disease or do you have something in mind?",,list("Make Your Own","Random","Choose")))
+			var/datum/round_event_control/event
+			switch(tgui_alert(usr,"What disease system do you want?",,list("Advanced","Simple","Make Your Own")))
```

Comment:
```suggestion
			var/prompt = tgui_alert(usr,"What disease system do you want?", "Disease Setup" ,list("Advanced","Simple","Make Your Own"))
			switch(prompt)
```

i think it's stupid to use this method to skip having a title and there's no real reason to _not_ have a title so let's just have a title here while we have the chance to fix it

Reply:
Sure, agreed it's nicer. I didn't even notice it was a switch there.